.github/workflows/deploy.yml

```diff
@@ -78,7 +78,7 @@ jobs:
             for i in 1 2 3 4; do [ $(curl --insecure --silent https://127.0.0.1/api/v1/version | jq .git_commit) = "\"${{ steps.prep.outputs.sha_short}}\"" ] && break || (if [ $i = 4 ]; then exit -1; else sleep 5; fi ) done 
             echo "All healthy!"
 
-      - if: github.event_name == 'release' || github.ref == 'refs/heads/main' || contains(github.event.head_commit.message, '[mainnetdeploy]')
+      - if: github.event_name == 'release' || contains(github.event.head_commit.message, '[mainnetdeploy]')
```

Comment:
This was incorrect since we don't want to ever deploy to mainnet when merging to main.
Deploying to mainnet should only happen on a release, or a correctly created commit message.